test/imgAutosize.js

```diff
@@ -37,8 +35,8 @@ describe('Plugin', () => {
 
         it('should autosize remote PNG', () => {
             return init(
-                '<img src="http://dummyimage.com/111x52/000/fff.png" width="auto" height="25" alt="hi">',
-                '<img src="http://dummyimage.com/111x52/000/fff.png" width="111" height="25" alt="hi">'
+                '<img src="http://placehold.it/200x200" width="auto" height="25" alt="hi">',
+                '<img src="http://placehold.it/200x200" width="200" height="25" alt="hi">'
```

Comment:
I guess it should be `width="25"` because the image is a square and height is 25 px.

Reply:
This was just based on the existing code that was there - should I make a new issue for this?

Reply:
Sorry, I just misread the param values. You're right, the bug was there already before your changes.

> should I make a new issue for this?

Yes, please.